Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -6752,5 +6752,16 @@ bool IWindowExplorer.TestElementLocators(ElementInfo EI, bool GetOutAfterFoundEl
             }
 
         }
+
+        public ObservableList<ElementLocator> GetLearningLocators()
```

Comment:
you done exactly what i asked but let's improve it even more:
1- add to PlatformInfoBase the GetLearningLocators() instead to IWindowExplorer and implement it in WebPlatformInfo.cs
2- in SeleniumDriver IWindowExplorer.GetElementLocators(ElementInfo ElementInfo), pull the locators list from WebPlatformInfo.cs, iterate over them and for each implement the actual Locate Value 
3- in POM Wizard also get the locators list from WebPlatformInfo.cs for your needs
this way Driver and POM Wizard UI always will be synced with Locators list and we won't need to update it in 2 seperate functions in SeleniumDriver as of now in current solution. 

IWindowExplorer.GetElementLocators(ElementInfo ElementInfo)

Reply:
Requested Changes-  …
-Clean & Smart logic for Getting Locators.
-missing prefix 'x' in control's name with x:Name
-added AutomationProperty.AutomationID for xLocatorsGrid for POM Unit Test in POMsPOM